samples/benchmarks/coremark/Kconfig

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menu "Zephyr Kernel"
+	source "Kconfig.zephyr"
+endmenu
+
+menu "CoreMark sample"
+
+config APP_MODE_FLASH_AND_RUN
+	bool "Run CoreMark benchmark on start up"
+	help
+	  If enabled, CoreMark will start execution immidietly after CPU start up.
+	  Otherwise, it will wait button to be pressed.
```

Comment:
That's a questionable option, especially given by default it will wait for a button which it's not easy to press in remote and automated setups. I might not be well aware of the current state of tests and samples in Zephyr these days, but I would imagine most if not all of them are meant to be run immediately after loading binary onto the target.

And why messing with a button at all? For measuring time? ;)
Let's make is as platform neutral as possible.